commit 1d798ab1d961a075c1391fd20b49940b05af07bb
Author: Orion Poplawski <orion@nwra.com>
Date:   Wed Oct 24 19:54:09 2012 -0600

    Update to 2.8.10 RC 3
    Rebase FindRuby and FindPostgreSQL patches

commit 0cf590138b7671c505d4d2800f85eabb40d175d7
Author: Orion Poplawski <orion@cora.nwra.com>
Date:   Mon May 21 10:39:01 2012 -0600

    Add patch to fix FindRuby (bug 822796)
diff -up cmake-2.8.10-rc1/Modules/FindRuby.cmake.findruby cmake-2.8.10-rc1/Modules/FindRuby.cmake
--- cmake-2.8.10-rc1/Modules/FindRuby.cmake.findruby	2012-10-02 10:12:18.000000000 -0600
+++ cmake-2.8.10-rc1/Modules/FindRuby.cmake	2012-10-02 15:50:30.463292214 -0600
@@ -92,14 +92,8 @@ if(RUBY_EXECUTABLE  AND NOT  RUBY_VERSIO
    _RUBY_CONFIG_VAR("sitearchdir" RUBY_SITEARCH_DIR)
    _RUBY_CONFIG_VAR("sitelibdir" RUBY_SITELIB_DIR)
 
-   # vendor_ruby available ?
-   execute_process(COMMAND ${RUBY_EXECUTABLE} -r vendor-specific -e "print 'true'"
-      OUTPUT_VARIABLE RUBY_HAS_VENDOR_RUBY  ERROR_QUIET)
-
-   if(RUBY_HAS_VENDOR_RUBY)
-      _RUBY_CONFIG_VAR("vendorlibdir" RUBY_VENDORLIB_DIR)
-      _RUBY_CONFIG_VAR("vendorarchdir" RUBY_VENDORARCH_DIR)
-   endif()
+   _RUBY_CONFIG_VAR("vendorlibdir" RUBY_VENDORLIB_DIR)
+   _RUBY_CONFIG_VAR("vendorarchdir" RUBY_VENDORARCH_DIR)
 
    # save the results in the cache so we don't have to run ruby the next time again
    set(RUBY_VERSION_MAJOR    ${RUBY_VERSION_MAJOR}    CACHE PATH "The Ruby major version" FORCE)
